cmd/kube-scheduler/app/server_test.go

```diff
@@ -147,6 +147,58 @@ profiles:
 		t.Fatal(err)
 	}
 
+	// policy config file
+	policyConfigFile := filepath.Join(tmpDir, "policy-config.yaml")
+	if err := ioutil.WriteFile(policyConfigFile, []byte(`{
+		"kind": "Policy",
+		"apiVersion": "v1",
+		"predicates": [
+		  {"name": "MatchNodeSelector"},
```

Comment:
I would just make this minimal. This is not a compatibility test. I just want to make sure the flag works.

Reply:
done